tests/topotests/evpn-pim-1/spine/bgp.summ.json

```diff
@@ -0,0 +1,46 @@
+{
+  "routerId":"192.168.100.1",
+  "as":65001,
+  "vrfId":0,
+  "vrfName":"default",
+  "tableVersion":7,
+  "peerCount":2,
+  "peers":{
+    "192.168.1.2":{
+      "hostname":"eva",
+      "remoteAs":65002,
+      "version":4,
+      "tableVersion":0,
+      "outq":0,
+      "inq":0,
+      "prefixReceivedCount":3,
+      "pfxRcd":3,
+      "pfxSnt":7,
+      "state":"Established",
+      "connectionsEstablished":1,
+      "connectionsDropped":0,
+      "idType":"ipv4"
+    },
+    "192.168.2.3":{
+      "hostname":"eva",
```

Comment:
@AnuradhaKaruppiah Can you just remove the hostname lines from this output

Reply:
sorry, missed this comment. will take care of it today.

Reply:
done